src/sidebar/components/search/SearchPannel.tsx

```diff
@@ -0,0 +1,54 @@
+import { Card, CardContent, CloseButton } from '@hypothesis/frontend-shared';
```

Comment:
Typo in filename - `SearchPannel`

Reply:
I have written `Pannel` with double n like a hundred times in the past few days 😅